### ISSUE TITLE
[passtheheadphones] Exception (passtheheadphones): Slow down! You may only log in once per minute. Last login: 2019-12-01 14:09:53: Slow down! You may only log in once per minute. Last login: 2019-12-01 14:09:53 (Config)

### ISSUE THREAD

**Issue by killerqueen619** [Jackett/Jackett#6461]
ss : http://prntscr.com/q4m5xn

**Comment by ThatNerdyPikachu** [comment]
RED is currently experiencing a credential stuffing attack (see blog), so this could be due to that. I'd give it a few days before trying again.

**Comment by garfield69** [comment]
this site uses the GazelleTracker service.
I noticed that although these GZ sites have a login page with the keeplogged option available, the jackett GZ abstract does not use it.
Which means that every time you make a query on these sites with Jackett, you may potentially be logging in every time.
So I've added the keeplogged to the login, and this may help reduce the login count and improve performance a tiny bit.
Or it make not make any difference at all, hard to tell.

try out jackett pre-release 0.12.1062.
other GZ sites are: Orpheus, CGPeers, notwhatcd, TehConnectionMe, Psytorrents, BrokenStones, HDOnly, AlphaRatio .
Note: I've only got an account on AlphaRatio so this GZ enhancement has only been tested on that site.

**Comment by ne0ark** [comment]
@garfield69 Still getting this error:

```
Jackett.Common.ExceptionWithConfigData: Slow down! You may only log in once per minute. Last login: 2019-12-15 03:34:21
   at Jackett.Common.Indexers.Abstract.GazelleTracker.<>c__DisplayClass17_0.<ApplyConfiguration>b__0() in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Abstract/GazelleTracker.cs:line 87
   at Jackett.Common.Indexers.BaseIndexer.ConfigureIfOK(String cookies, Boolean isLoggedin, Func`1 onError) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/BaseIndexer.cs:line 267
   at Jackett.Common.Indexers.Abstract.GazelleTracker.ApplyConfiguration(JToken configJson) in /home/appveyor/projects/jackett/src/Jackett.Common/Indexers/Abstract/GazelleTracker.cs:line 93
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/IndexerApiController.cs:line 110
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.UsePathBaseMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/appveyor/projects/jackett/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 73``` on redacted.

**Comment by garfield69** [comment]
then its beyond my skills to fix.
Ping to any c# developers that can help sort this one out.

**Comment by m4nnerheim** [comment]
I'm still encountering this issue

**Comment by ngosang** [comment]
I don't have an account to fix and test it. If someone have one send me.

**Comment by graydonpleasants** [comment]
can verify this does not fix the error on redacted using version:
`v0.12.1605-ls39`

**Comment by SawkeeReemo** [comment]
Checking in... Setting up Jacket for the first time tonight, and I, too, am running into this issue with Redacted. Any updates? (I'll check in the Redacted forum as well and report back here if I find anything.)

**Comment by ngosang** [comment]
None of the current developers have account in Redacted (PassTheHeadphones) so we can't help.
Maybe you can invite us or ask a site administrator for an invitation for us.
@RoloSoze you are in?

**Comment by RoloSoze** [comment]
> @RoloSoze you are in?

No.

**Comment by garfield69** [comment]
Jackett 0.14.164

**Comment by SawkeeReemo** [comment]
> None of the current developers have account in Redacted (PassTheHeadphones) so we can't help.
> Maybe you can invite us or ask a site administrator for an invitation for us.
> @RoloSoze you are in?

@ngosang 
I may be able to help you get in, I know someone in there. But I need a better way to talk to you about this than a public comment thread. Any suggestions? (I looked for a private message system here, but I'm fairly new to GitHub and didn't see one when I quickly looked.)

**Comment by ngosang** [comment]
@SawkeeReemo thank you but someone else already invited me.
This issue is fixed in Jackett >= 0.14.164
